gitpod-io/gitpod
# [billing] provide human readable price info

## Description
Use a human-readable string configured on the stripe price to inform the user of their current subscription.
Aligns prices between teams and individuals, so that teams get the first 1000 credits for 9 USD as well.

<img width="611" alt="Screenshot 2023-01-19 at 17 49 51" src="https://user-images.githubusercontent.com/372735/213513735-7ff23a29-16c7-4bac-8341-8099583ee78e.png">

## Related Issue(s)

Fixes #15786

Depends on
- https://github.com/gitpod-io/ops/issues/7904

Also related 
 - https://github.com/gitpod-io/ops/pull/7799
 - https://github.com/gitpod-io/ops/pull/7800

## How to test

Check subscription for a new team is $9/month and verify the information presented makes sense before and after subscribing.

## Release Notes

```release-note
Teams get the first 1000 credits for 9 USD/€
```

## Documentation


## Build Options:

      Experimental feature to run the build with GitHub Actions (and not in Werft).
      If enabled this will build `install/preview`
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
